insights/parsers/tests/test_ps.py

```diff
@@ -377,7 +377,9 @@ def test_ps_eo():
+ 2365 ?        Ssl    8:52 /usr/bin/docker-containerd-current -l unix:///var/run/docker/libcontainerd/docker-containerd.sock --shim docker-containerd-shim --metrics-interval=0 --start-timeout 2m --state-dir /var/run/docker/libcontainerd/containerd --runtime docker-runc --runtime-args --systemd-cgroup=true REGISTRIES= --add-registry registry.access.redhat.com OPTIONS=--selinux-enabled --log-driver=journald --signature-verification=false DOCKER_HTTP_HOST_COMPAT=1 PATH=/usr/libexec/docker:/usr/bin:/usr/sbin PWD=/ GOTRACEBACK=crash LANG=en_US.UTF-8 DOCKER_NETWORK_OPTIONS= SHLVL=0 DOCKER_CERT_PATH=/etc/docker DOCKER_STORAGE_OPTIONS=
+24652 pts/2    R+     0:00 ps aexww XDG_VTNR=2 XDG_SESSION_ID=2 XDG_CURRENT_DESKTOP=GNOME COLORTERM=truecolor BASH_FUNC_module()=() {  eval `/usr/bin/modulecmd bash $*` } BASH_FUNC_scl()=() {  local CMD=$1;  if [ "$CMD" = "load" -o "$CMD" = "unload" ]; then  eval "module $@";  else  /usr/bin/scl "$@";  fi } _=/usr/bin/ps
```

Comment:
https://github.com/RedHatInsights/insights-core/pull/1866#issuecomment-486551320

And another concern is we are not sure if there will be more complicated syntax appearances in the ENV list. So we try to revert the `ps_aexww`